.travis.yml

```diff
@@ -3,18 +3,10 @@ cache: pip
 dist: trusty
 python:
   - "3.6"
-addons:
-  apt:
-    sources:
-      - boost-latest
-      - ubuntu-toolchain-r-test
-    packages:
-      - libboost-system1.55-dev
-      - libboost-filesystem1.55-dev
-      - g++-4.8
+  - "3.7"
 
 before_install:
-  - pip install pip --upgrade
+  - pip install pip pytest --upgrade
 
 install:
   # annoyingly we can't put this in setup.py...
```

Comment:
@znicholls Can we get rid of the dependency on https://github.com/thedataincubator/expectexception ? With unit tests we don't need testing for exceptions in notebooks or do we?

Reply:
It's for allowing cells to fail with an exception in a notebook.

Reply:
Yes, but why do we need that?

Reply:
(I would just like to get rid of dependencies on repositories rather than proper packages)

Reply:
> Yes, but why do we need that?

To show to users how things can go wrong and which errors to expect?

> (I would just like to get rid of dependencies on repositories rather than proper packages)

I think the usual solution here is to vendor the code, i.e. just include it in our repo.
Since it is only needed in the notebooks you could just copy the module folder to our notebooks directory and also copy license and their Readme file into it.

Otherwise i think it's not much different having somthing on GitHub vs. PyPI (you could pin the dependency to a specific commit) - both could disappear.

Reply:
Hmm, ok, I will have a look and maybe just use a `try ... except` for that purpose...

Reply:
> Hmm, ok, I will have a look and maybe just use a try ... except for that purpose...

Which purpose? Not sure i understand.